### DISCUSSION TITLE
【灌水】为什么有的猫希人更像猫

### DISCUSSION THREAD

**Discussion by LEOPard824** [share121/inter-knot#1945] [其他]
我对这个问题好好奇，有没有好心绳匠能够解答
![20240813220046](https://github.com/user-attachments/assets/83744218-f58b-4327-9cfa-4345616ff123)
![20240813215815](https://github.com/user-attachments/assets/7c929dfe-e497-4b26-a28a-b81a1208527b)

**Comment by yys20071108** [comment]
因为本来就是猫变化过来的

**Comment by 566001** [comment]
因为猫在第一个字

**Comment by yixiaomoya** [comment]
万能回答:个体差异
  
参考来源--某位猫希人:
人与人的体质是不同的,我曾在极度愤怒的情况下愤怒了1天
参考来源--某位熊希人
我不小心被大运创飞10来米远,还好我是体育生,站起来拍拍灰就没事了

**Comment by hfwang132** [comment]
因为要满足各种程度的福瑞控的需要~
